vendor/github.com/ceph/ceph-csi/api/deploy/kubernetes/cephfs/csi-config-map.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```

Comment:
This file needs `---` on the 1st line like all others, `yamllint` complains about it too:

```
  8:1       warning  missing document start "---"  (document-start)
```

Reply:
Thanks for the review Niels, updated.